### ISSUE TITLE
Images are Inaccessible to Screen Readers

### ISSUE THREAD

**Issue by aliviahossain** [GSSoC-Flow/WardhaMetroFLow#141]
Many ImageView and ImageButton elements in the app are missing a contentDescription. This attribute provides a text description of the image's content or function. Without it, screen readers like Android's TalkBack can't describe the image to a user, simply announcing it as "unlabeled button" or ignoring it completely.

PLEASE ASSIGN ME THIS ISSUE UNDER THE GSSOC' 25 LABEL. THANK YOU.
